.github/workflows/cron-goodbits-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Cron sync goodbits list
+
+on:
+  schedule:
+    - cron: '13 0,5,10,15,20 * * *'
```

Comment:
is this "every 5 hours at 13 mins past" to avoid the rush of doing things on the hour?

Reply:
Kept same timings as we did for denylist https://github.com/nftstorage/nft.storage/pull/1761

We can revisit later if we need more/less frequency. No strong opinions for now.